pkg/cortex/modules.go

```diff
@@ -425,7 +427,10 @@ func (t *Cortex) initTableManager() (services.Service, error) {
 		os.Exit(1)
 	}
 
-	tableClient, err := storage.NewTableClient(lastConfig.IndexType, t.Cfg.Storage)
+	reg := prometheus.WrapRegistererWith(
+		prometheus.Labels{"component": "table-manager-index-chunk-" + lastConfig.From.String()}, prometheus.DefaultRegisterer)
```

Comment:
I would say `table-manager-store` as label value would be enough, being the table manager used to manage the index/chunks store, differently then the `table-manager-delete-requests-store` below.

Reply:
We want to change table manager to manage table for older schemas as well so I think we would have to add back the start time of the schema then which we might miss so I added it now itself. I can change just the prefix to `table-manager-store-` if you don't mind.

Reply:
I'm always in favour of doing a change when required (right now, it's not and if in the future you change the value of `component` it's not even considered a CHANGE, contrary to adding a new label like in this case) but **I'm not opposed** if you prefer to already do it. Up to you.

Reply:
Fair enough changed it. Thanks!